mmdet/models/anchor_heads/anchor_head.py

```diff
@@ -84,6 +88,14 @@ def __init__(self,
         self.loss_bbox = build_loss(loss_bbox)
         self.train_cfg = train_cfg
         self.test_cfg = test_cfg
+        if self.train_cfg:
+            self.assigner = build_assigner(self.train_cfg.assigner)
+            # use PseudoSampler when sampling is False
+            if self.sampling and hasattr(self.train_cfg, 'sampler'):
+                sampler_cfg = self.train_cfg.sampler
```

Comment:
If `self.sampling` is True, but `self.train_cfg` has no field `sampler`, should we raise an exception?